src/components/index-page/IndexExasperation.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import decorativeCircle from "../../assets/images/decorative-circle.png"
+import "./css/index-exasperation.css"
+
+export const IndexExasperation = () => {
+  return (
+    <div className="index-exasperation">
+      <div className="index-exasperation__contents">
+        <img className="index-exasperation__decorative-circle" src={decorativeCircle} />
```

Comment:
for accessibility, given this image is purely presentational we should set `alt=""` and `role="presentation"`